models/api_rate.go

```diff
@@ -65,10 +65,6 @@ func (r *Rate) SpecialServicesRequested() *SpecialServicesRequested {
 func (r *Rate) Weight() Weight {
 	commoditiesWeight := r.Commodities.Weight()
 	if !commoditiesWeight.IsZero() {
-		// Assume the weight must be between than 13 and 150 lbs.
-		// If the weight is less than 13 lbs, assume a weight of 13 lbs, which is
-		// heavy enough that the destination will matter when choosing between two
-		// fedex ground rates
 		commoditiesWeight.Value = math.Min(commoditiesWeight.Value, 150.0)
```

Comment:
curious where did you define 13oz as the default when weight is not available at any place as mentioned in AC3 
> if no default weights available or if item count is unavailable, 13oz should be used for the entire return package in rate call

Reply:
At the end of the method:
```

func (r *Rate) Weight() Weight {
	commoditiesWeight := r.Commodities.Weight()
	if !commoditiesWeight.IsZero() {
		commoditiesWeight.Value = math.Min(commoditiesWeight.Value, 150.0)
		return commoditiesWeight
	}

	return Weight{Units: WeightUnitsLB, Value: 13}
}
```

Reply:
@llacerdapaypal I think there is a misunderstanding between us. I am trying to find implementation for expected behavior 3
![image](https://github.com/happyreturns/fedex/assets/94640719/aebe6922-4aad-44d1-911d-c2e6b7226e85)

Reply:
[About the list check the activity section](https://happyreturns.atlassian.net/browse/HUB-3702)
1. if weight of items is known, the sum of all item weights in the return should be used for rate call
![Screenshot 2023-12-01 at 9 52 24 AM](https://github.com/happyreturns/fedex/assets/146731273/d431b6a1-1f60-45e7-8b4b-15701f1c1a0a)
2. if weights of items are not known, the sum of the default weights configured by retailer in Dashboard should be used for rate call.
![Screenshot 2023-12-04 at 10 29 01 AM](https://github.com/happyreturns/fedex/assets/146731273/d5502114-e70c-465e-aa05-4c46af94c9eb)
3.if no default weights available or if item count is unavailable, 13oz should be used for the entire return package in rate call
Applied in the last commit.